### DISCUSSION TITLE
turbo breakdown

### DISCUSSION THREAD

**Discussion by jnson-Li** [vercel/turborepo#9172] [Help] [answered]
### Summary

I hope you can help me fix this mistake
![20240920-141301](https://github.com/user-attachments/assets/3dda0015-9633-4852-bf7f-b9fc64ce8021)

### Additional information

```js
"turbo": "1.13.4"

 "packageManager": "pnpm@9.1.1",
  "engines": {
    "node": ">=18"
  }


node   v20.14.0
```

**Comment by anthonyshew** [comment, marked as answer]
Hi, could you report an issue following our bug template? https://github.com/vercel/turborepo/issues/new?assignees=&labels=kind%3A+bug%2Cneeds%3A+triage&projects=&template=0-turborepo-bug-report.yml

Thank you!
